test-framework/jacoco/deployment/src/main/java/io/quarkus/jacoco/deployment/JacocoProcessor.java

```diff
@@ -57,6 +57,7 @@ void transformerBuildItem(BuildProducer<BytecodeTransformerBuildItem> transforme
                 + config.dataFile;
         System.setProperty("jacoco-agent.destfile",
                 dataFile);
+        System.setProperty("jacoco-agent.jmx", "true");
```

Comment:
This doesn't seem related? I just want to confirm you want this in.

Reply:
I actually added the same comment, but I think I never published it...

Reply:
Oops, that was part of something else I was playing with